From 86e793c7264900f39f706eb8862d4c028b242d7b Mon Sep 17 00:00:00 2001
From: Dirkjan Ochtman <dirkjan@ochtman.nl>
Date: Mon, 30 Oct 2023 21:24:45 +0100
Subject: [PATCH 10/12] Don't use workspace-wide crate versions (#184)

Given that rustls-cert-gen already specifies its version number (such
that only one crate actually uses the workspace `version`) and that
version numbers are generally pretty closely tied to specific crate's
API, IMO it's better to avoid having a workspace-level crate version.
---
 Cargo.toml       | 1 -
 rcgen/Cargo.toml | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -9,7 +9,6 @@ pem = { version = "3.0.2" }
 license = "MIT OR Apache-2.0"
 edition = "2021"
 readme = "README.md"
-version = "0.11.3"
 description = "Rust X.509 certificate generator"
 repository = "https://github.com/rustls/rcgen"
 keywords = ["mkcert", "ca", "certificate"]
--- a/rcgen/Cargo.toml
+++ b/rcgen/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "rcgen"
+version = "0.11.3"
 documentation = "https://docs.rs/rcgen"
-version.workspace = true
 description.workspace = true
 repository.workspace = true
 readme.workspace = true
